api/README.md

```diff
@@ -0,0 +1,215 @@
+
+
+# api.mstacm.org
+[View Live](https://api.mstacm.org) |
+[Report Bug](https://github.com/sigdotcom/mstacm.org/issues) |
+[Request Feature](https://github.com/sigdotcom/mstacm.org/issues)
+
+
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
```

Comment:
But they are still licensed individually even if they're part of the monorepo because they're separate packages. In theory we can have packages of different licenses.